unlock-app/src/hooks/useAuthenticate.ts

```diff
@@ -159,5 +159,6 @@ export function useAuthenticate() {
     signInWithPrivy,
     signOut,
     privyReady,
+    isLoading: !privyReady || (privyAuthenticated && !wallets[0]?.address),
```

Comment:
Should we also do that when there is a wallet BUT we don't have locksmith token ready?